packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -95,6 +95,12 @@ exports.questions = {
 		fieldName: 'letters-to-neighbours',
 		validators: [new RequiredValidator()]
 	}),
+	uploadLettersToNeighbours: new MultiFileUploadQuestion({
+		title: 'Uploaded letters',
+		question: 'Upload the letters and emails',
+		fieldName: 'upload-letters-emailss',
```

Comment:
We can now use the new html property from AAPD-301 in here. Main suggestion would be to use a naming convention like resources/upload-letters-emails/content.html for the location of the html